### DISCUSSION TITLE
useFieldArray input does not clear errors set via setError

### DISCUSSION THREAD

**Discussion by asaf** [react-hook-form/react-hook-form#5306] [Q&A] [answered]
**Describe the bug**

`useFieldArray` with errors set via `setError` (in submit method) does not clear error after value is modified.

**Codesandbox link (Required)**
https://codesandbox.io/s/calc-forked-kkk7d?file=/src/App.tsx:1042-1075

V7

**Comment by bluebill1049** [comment, marked as answer]
https://react-hook-form.com/api/useform/seterror

<img width="930" alt="Screen Shot 2021-05-24 at 7 05 48 pm" src="https://user-images.githubusercontent.com/10513364/119324344-0e549180-bcc3-11eb-9231-2eb0c12f3056.png">

> An error that is not associated with an input field will be persisted until cleared with clearErrors.

**Reply by asaf** [reply]
@bluebill1049 thanks for responding,

I see, error key should be `cart.0.name` instead of `cart.0`, thanks!
